scripts/generateAssetData/generateRelatedAssetIndex/mapping.ts

```diff
@@ -0,0 +1,59 @@
+import type { AssetId, ChainId } from '@shapeshiftoss/caip'
+import {
+  arbitrumChainId,
+  ASSET_NAMESPACE,
+  avalancheChainId,
+  bscChainId,
+  ethChainId,
+  optimismChainId,
+  polygonChainId,
+  toAssetId,
+} from '@shapeshiftoss/caip'
+
+import type { ZerionImplementation } from './validators/fungible'
+
+export const ZERION_CHAINS = [
+  // shapeshift supported
+  'avalanche',
+  'binance-smart-chain',
+  'ethereum',
+  'optimism',
+  'polygon',
+  'arbitrum',
+  // not yet
+  // 'aurora',
+  // 'fantom',
+  // 'solana',
+  // 'xdai',
```

Comment:
Are we in a state where we can now enable this, or is this this not enabled upstream?

Reply:
<img width="392" alt="image" src="https://github.com/shapeshift/web/assets/17035424/e7f9726e-3e0f-4df4-baff-b9fdc2271e1a">

Lools ike Gnosis chain support is properly enabled upstream

Reply:
This was a legit issue, fixed in e0ebc7869f19cb4eb09a0ce49f62c755fda42ec2